packages/at_onboarding_flutter/lib/screen/at_onboarding_backup_screen.dart

```diff
@@ -25,116 +38,147 @@ class AtOnboardingBackupScreen extends StatefulWidget {
-            Center(
-              child: Image.asset(
-                AtOnboardingStrings.backupZip,
-                height: Platform.isAndroid || Platform.isIOS
-                    ? MediaQuery.of(context).size.height * 0.3
-                    : 250,
-                width: Platform.isAndroid || Platform.isIOS
-                    ? MediaQuery.of(context).size.height * 0.3
-                    : 250,
-                fit: BoxFit.fill,
-                package: AtOnboardingConstants.package,
-                color: Platform.isIOS || Platform.isAndroid
-                    ? Theme.of(context).brightness == Brightness.light
-                        ? Colors.black
-                        : Colors.white
-                    : null,
+        body: Padding(
+          padding: const EdgeInsets.all(AtOnboardingDimens.paddingNormal),
+          child: Column(
+            // mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: <Widget>[
+              const SizedBox(height: 10),
+              const Text(
+                AtOnboardingStrings.saveImportantTitle,
+                style: TextStyle(
+                    fontSize: AtOnboardingDimens.fontLarge,
+                    fontWeight: FontWeight.bold),
+                textAlign: TextAlign.center,
               ),
-            ),
-            Expanded(flex: 1, child: Container()),
-            Container(
-              constraints: const BoxConstraints(
-                maxWidth: 400,
+              const SizedBox(height: 10),
+              const Text(
+                AtOnboardingStrings.saveBackupDescription,
+                style: TextStyle(fontSize: AtOnboardingDimens.fontNormal),
+                textAlign: TextAlign.center,
               ),
-              child: AtOnboardingPrimaryButton(
-                height: 48,
-                borderRadius: 24,
-                child: Text(
-                  AtOnboardingStrings.saveButtonTitle,
-                  style: TextStyle(
-                    color: Platform.isIOS || Platform.isAndroid
-                        ? Theme.of(context).brightness == Brightness.light
-                            ? Colors.white
-                            : Colors.black
-                        : null,
-                  ),
+              Expanded(flex: 1, child: Container()),
+              Center(
+                child: Image.asset(
+                  AtOnboardingStrings.backupZip,
+                  height: Platform.isAndroid || Platform.isIOS
+                      ? MediaQuery.of(context).size.height * 0.3
+                      : 250,
+                  width: Platform.isAndroid || Platform.isIOS
+                      ? MediaQuery.of(context).size.height * 0.3
+                      : 250,
+                  fit: BoxFit.fill,
+                  package: AtOnboardingConstants.package,
+                  color: Platform.isIOS || Platform.isAndroid
```

Comment:
What will be color for  desktop platforms ?

Reply:
Sorry, I forgot about it.  I've fixed and pushed code. Could you check again? @sachins-geekyants